task/banking.ts

```diff
@@ -0,0 +1,164 @@
+// defining interfaces.
+interface User {
+    id: number;
+    name: string;
+    email: string;
+    password: string;
+}
+
+interface Transaction {
+    id: number;
+    userId: number;
+    amount: number;
+    type: 'withdrawal' | 'deposit' | 'transfer';
+}
+
+interface Account {
+    id: number;
+    userId: number;
+    balance: number;
+}
+
+// defining class for different operations
+class Bank {
+    private users: User[] = [];
+    private transactions: Transaction[] = [];
+    private accounts: Account[] = [];
+
+    createUser(user: User): void {
+        const newUser: User = {
+            id: user.id,
+            name: user.name,
+            email: user.email,
+            password: user.password,
+        };
+        this.users.push(newUser);
```

Comment:
In the `createUser` method, you can directly push the `user` parameter into the user's array instead of creating a new object newUser.